worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -7,48 +7,49 @@
-4. Insert the string of the depot you wish to download. For the AP-supported v1.15, you will want to use: `download_depot 374320 374321 4471176929659548333`.
-5. Steam will now download the depot. Note: There is no progress bar for the download in Steam, but it is still downloading in the background.
-6. Back up your existing game executable (`DarkSoulsIII.exe`) found in `\Steam\steamapps\common\DARK SOULS III\Game`. Easiest way to do this is to move it to another directory. If you have file extensions enabled, you can instead rename the executable to `DarkSoulsIII.exe.bak`.
-7. Return to the Steam console. Once the download is complete, it should say so along with the temporary local directory in which the depot has been stored. This is usually something like `\Steam\steamapps\content\app_XXXXXX\depot_XXXXXX`.
-8. Take the `DarkSoulsIII.exe` from that folder and place it in `\Steam\steamapps\common\DARK SOULS III\Game`.
-9. Back up and delete your save file (`DS30000.sl2`) in AppData. AppData is hidden by default. To locate it, press Windows Key + R, type `%appdata%` and hit enter. Alternatively: open File Explorer > View > Hidden Items and follow `C:\Users\<your_username>\AppData\Roaming\DarkSoulsIII\<numbers>`.
-10. If you did all these steps correctly, you should be able to confirm your game version in the upper-left corner after launching Dark Souls III.
+1. Before you first connect to a multiworld, run `randomizer\DS3Randomizer.exe`.
 
+2. Put in your Archipelago room address (usually something like `archipelago.gg:12345`), your player
+   name (also known as your "slot name"), and your password if you have one.
 
-## Installing the Archipelago mod
+3. Click "Load" and wait a minute or two.
 
-Get the `dinput8.dll` from the [Dark Souls III AP Client](https://github.com/Marechal-L/Dark-Souls-III-Archipelago-client/releases) and 
-add it at the root folder of your game (e.g. `SteamLibrary\steamapps\common\DARK SOULS III\Game`)
+### Running and Connecting the Game
 
-## Joining a MultiWorld Game
+To run _Dark Souls III_ in Archipelago mode:
 
-1. Run Steam in offline mode to avoid being banned.
-2. Launch Dark Souls III.
-3. Type in `/connect {SERVER_IP}:{SERVER_PORT} {SLOT_NAME} password:{PASSWORD}` in the "Windows Command Prompt" that opened. For example: `/connect archipelago.gg:38281 "Example Name" password:"Example Password"`. The password parameter is only necessary if your game requires one.
-4. Once connected, create a new game, choose a class and wait for the others before starting.
-5. You can quit and launch at anytime during a game.
+1. Start Steam. **Do not run in offline mode.** The mod will make sure you don't connect to the
+   DS3 servers, and running Steam in offline mode will make certain scripted invaders fail to spawn.
 
-## Where do I get a config file?
+2. Run `launchmod_darksouls3.bat`. This will start _Dark Souls III_ as well as a command prompt that
+   you can use to interact with the Archipelago server.
+
```

Comment:
Considering the last version worked on linux, a linux alternative should be provided for this batch file, and should be mentioned in these instructions

Reply:
Trying it out, it doesn't work because it looks for the path to the steam library in the windows registry... It's also been discontinued in favour of me3, but that in turn looks a bit too immature to start using already? Alternatively, it looks vaguely abandoned, judging by the commit history... Not sure what I think we should do, but relying on discontinued or abandoned software doesn't sound wise. An archipelago fork of ME2?